### PR TITLE
Ports monke GAGS, and more QoL [WIP]

### PR DESCRIPTION
**Pull request by Tsar-Salat**
Ports:
- https://github.com/Monkestation/MonkeStation/pull/608/

## About The Pull Request

This pr ports several features, including:
- Vending Machine tabs
- GAGs most visually similiar clothing
- Spraycans can apply gags color
- tape can now be used as an alternative to a muffler, to cover someones mouth 
- adds new tot tape to utilize the muffle function and more
- made all usages of "camoflague", "camoflage" "camouflage" match because it was pissing me off
- adds kimono, gaggable dress
- reorganizes bitflags for GAGS
- edited some beret armor to make more sense

## Why It's Good For The Game
Greyscaling duplicate icons is a great way to save space, as assigning a color to an icon is way more efficient. Also, doing this to basically every clothing item we can is an easy way to increase character customization, as basically any greyscale hexcode can be customized ingame, once we actually port the needed stuff into vendors.

The additional outfit added (kimono) is also a good demonstration of how GAGS customization can work (and it goes thematically with the other Japanese stuff we have)

Before this pr, the muffler found in the re-education chamber was kinda the only way to muffle people, this adds a new ghetto method in the form of tape. Tape can already be found in maints and should be just as easily craftable as the other SAW crap like cable cuffs and such IMO

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl: dwasint, RKz
add: More GAGs clothing
add: Kimono, Japanese-style GAGable clothing
add: tape can now be used as a ghetto muffler to tape someone's mouth shut. MMMMPH!
add: a new type of traitor tape, it sticks to people and hurts when you pull it off someone's mouth! Spiky!
code: reorg bitflags
spelling: kills the awful "camoflague"

/:cl:

### CODE REVIEW

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -722,87 +856,134 @@ GLOBAL_LIST_EMPTY(vending_products)
...
+	return TRUE
 
-				// each department (seller_department) will earn the profit
-				if(price_to_use && seller_department)
-					var/list/dept_list = SSeconomy.get_dept_id_by_bitflag(seller_department)
-					if(length(dept_list))
-						price_to_use = round(price_to_use/length(dept_list))
-						for(var/datum/bank_account/department/D in dept_list)
-							if(D)
-								D.adjust_money(price_to_use)
```
Comment (EvilDragonfiend) [inline code/modules/vending/_vending.dm:786]: you're removing the recent bank system change.

**Comment by EvilDragonfiend** [comment]
I am also touching vendor system in my PR. It's tedious to see when PRs are touching the same file lol

File: code/modules/vending/_vending.dm (hunk 1 above)
Comment (Tsar-Salat) [reply code/modules/vending/_vending.dm:786]: yeah, it kept causing conflicts. I cut it out for the time being so I could work out the icon runtimes first

**Comment by Tsar-Salat** [comment]
> I am also touching vendor system in my PR. It's tedious to see when PRs are touching the same file lol

If you include vendor tabs and GAGs recoloring in your pr, I will gladly cut vendor related stuff out of this one, if that helps

**Comment by Tsar-Salat** [comment]
> I am also touching vendor system in my PR.

ah, lmfao. Me messing up the economy fixes of the vendor file is more my incompetence with economy code than anything else. You certainly made it much more solid than before haha

On another note, are you supportive of the categories, personally? 

I would like to minimize conflicts, but I am definitely a bit attached to vendor categories

**Comment by EvilDragonfiend** [comment]
I guess the vendor one can be revisited later. I will check once the base GAGs system is ported then try to adjust the vending one.

**Comment by Tsar-Salat** [comment]
alright, reverting all vending-related commits to reduce conflicts with Dragon's pr

**Pull request closed without merging**